code/game/objects/items/weapons/RCL.dm

```diff
@@ -24,18 +25,35 @@
 	if(istype(W,/obj/item/stack/cable_coil))
 		if(!loaded)
 			loaded = W
+			loaded.max_amount = max_amount //We store a lot.
 			user.drop_item(W,src)
 		else
 			loaded.preattack(W,user,1)
 		update_icon()
 		user << "<span class='notice'>You add the cables to the [src]. It now contains [loaded.amount].</span>"
+	else if(isscrewdriver(W))
+		if(!loaded) return
+		user << "<span class='notice'>You loosen the securing screws on the side, allowing you to lower the guiding edge and retrieve the wires.</span>"
+		while(loaded.amount>30) //There are only two kinds of situations: "nodiff" (60,90), or "diff" (31-59, 61-89)
+			var/diff = loaded.amount % 30
+			if(diff)
+				loaded.use(diff)
+				getFromPool(/obj/item/stack/cable_coil,user.loc,diff)
+			else
+				loaded.use(30)
+				getFromPool(/obj/item/stack/cable_coil,user.loc,30)
+		loaded.max_amount = initial(max_amount)
+		loaded.loc = user.loc
+		user.put_in_hands(loaded)
+		loaded = null

```

Comment:
I'm not going to bother analyzing this too heavily, but I hope you've tested it so that it doesn't inappropriately disappear cables coils for you or whatever.